tensorflow/tsl/protobuf/dnn.proto

```diff
@@ -174,3 +174,9 @@ message ConvolutionDescriptorProto {
   // Tensorflow node name, same as in NodeDef, for debugging purposes.
   string name = 7;
 }
+
+// FusedMHAKind kind
+enum FusedMHAKind {
+  BMM1_OUTPUT_INPUT_TYPE = 0;
```

Comment:
Best practice is to add a default unknown entry, so this will become:

  BMM1_OUTPUT_UNKNOWN = 0;
  BMM1_OUTPUT_INPUT_TYPE = 1;
  BMM1_OUTPUT_FLOAT = 2;